### DISCUSSION TITLE
AuditFile.SourceDocuments.SalesInvoices.7Invoice (NC S2021/1) contém linha de crédito, por favor verifique.

### DISCUSSION THREAD

**Discussion by Edson2001** [assoft-portugal/SAF-T-AO#122] [Q&A] [answered]
Como posso resolver?

**Comment by cryptolopes** [comment, marked as answer]
Deve existir uma linha com `CreditAmount`, quando apenas são esperadas linhas com `DebitAmount`.

**Comment by Edson2001** [comment]
Obrigado pela resposta, resolveu.

Em qui, 4 de fev de 2021 20:08, Nelson <notifications@github.com> escreveu:
